src/services/adherence.service.ts

```diff
@@ -142,8 +142,39 @@ async function getAdherenceAlerts(
   return result.data
 }
 
+async function markAdherenceAlertsAsRead(
+  studyId: string,
+  alertIds: string[],
+  token: string
+): Promise<any> {
+  const endpoint = constants.endpoints.adherenceAlertsMarkRead.replace(':studyId', studyId)
+  const result = await Utility.callEndpoint(
+    endpoint,
+    'POST',
+    {studyId: studyId, alertIds: alertIds},
+    token
+  )
+  return result
+}
+
+async function markAdherenceAlertsAsUnread(
+  studyId: string,
+  alertIds: string[],
+  token: string
+): Promise<any> {
+  const endpoint = constants.endpoints.adherenceAlertsMarkUnread.replace(':studyId', studyId)
+  const result = await Utility.callEndpoint(
+    endpoint,
+    'POST',
+    {studyId: studyId, alertIds: alertIds},
```

Comment:
it's confusing in the documentation but I think you only need alertIds since you are passing a studyId in the url